students/kulon_aleksander/lesson_10/filling_in_the_gaps.py

```diff
@@ -19,11 +19,9 @@ def main():
     sorted_numbers = sorted(sorted_numbers)
 
     for i in range(int(sorted_numbers[0]), int(sorted_numbers[-1]) - 1):
-        current = str(sorted_numbers[i-1]).zfill(3)
-        next = str(int(sorted_numbers[i-1]) + 1).zfill(3)
-        later = str(int(sorted_numbers[i-1]) + 2).zfill(3)
+        next = str(int(sorted_numbers[i - 1]) + 1).zfill(3)
```

Comment:
Try to avoid built-in functions name. ```next``` is one of them https://docs.python.org/3.6/library/functions.html
Just remember about this next time 😄